build_debian.sh

```diff
@@ -260,6 +260,10 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
 ## Prepare empty directory to trigger mount move in initramfs-tools/mount_loop_root, implemented by patching
 sudo mkdir $FILESYSTEM_ROOT/host
 
+## Add /var/log mount entry to fstab
+## Loop device itself is created by onie installer
+sudo LANG=C chroot $FILESYSTEM_ROOT bash -c 'echo "/host/disk-img/var-log.ext4 /var/log ext4 rw,loop 0 0" >> /etc/fstab'
```

Comment:
We cannot access files in root filesystem during installation, so I put it in build_debian